core/modules/main/controllers/Main.php

```diff
@@ -2778,7 +2778,7 @@ public function runMarkAsBlocker(framework\Request $request)
      */
     public function runDeleteIssue(framework\Request $request)
     {
-        $request_referer = ($request['referer'] ?: isset($_SERVER['HTTP_REFERER']) ? $_SERVER['HTTP_REFERER'] : null);
+        $request_referer = isset($request['referer']) ? urldecode(html_entity_decode($request['referer'])) : (isset($_SERVER['HTTP_REFERER']) ? $_SERVER['HTTP_REFERER'] : null);
```

Comment:
It decoded here since value of 'referer' is encoded (I did it) in _issuemoreactions.inc.php
If not encoded, value referer will splited into many since referer will contain value like '&'.